src/middlewared/middlewared/etc_files/nsswitch.conf

```diff
@@ -12,7 +12,7 @@
 
         ad_enabled = safe_call('activedirectory.config')['enable']
         ldap_enabled = safe_call('ldap.config')['enable']
-        nis_enabled = safe_call('nis.config')['enable']
+        nis_enabled = safe_call('nis.config')['enable'] if IS_FREEBSD else False
```

Comment:
No need to change: `IS_FREEBSD and safe_call('nis.config')['enable']` like did below